src/main/java/edu/ucsb/cs156/happiercows/controllers/UserCommonsController.java

```diff
@@ -89,6 +89,7 @@ public ResponseEntity<String> putUserCommonsByIdBuy(
 
         if(userCommons.getTotalWealth() >= commons.getCowPrice() ){
           userCommons.setTotalWealth(userCommons.getTotalWealth() - commons.getCowPrice());
+          userCommons.setCowHealth((userCommons.getCowHealth()*userCommons.getNumOfCows() + 100) / (userCommons.getNumOfCows() + 1));
```

Comment:
No... we don't want to do this.  Instead, there is a job that will update cow health periodically.

There may be some old documentation that suggests this is the way to do it, but that's no longer how we are doing it.   Please leave cow health unchanged when we buy/sell cows.